markdown_stoplight/use-cases.md

```diff
@@ -1,128 +1,157 @@
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
-|`GET`|[/products/{productId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
-|`GET`|[/order/{orderId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
-|`GET`|[/orders/{orderId}/processes/shippings](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
+|`GET`|[/orders](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list)
+|`GET`|[/products/{productId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/product-get)
+|`GET`|[/order/{orderId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
+|`GET`|[/orders/{orderId}/processes/shippings](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-processes) |`GET`|[/orders/{orderId}/processes/shippings/{shippingProcessId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-shipping-process)
 
 ### Accounting
 
-Enable merchants to track payments more easily, spot unpaid orders, or quickly evaluate the performance of their shop. You can offer notifications for payments that have been pending for too long or monitor single orders. Aggregating the turnover from orders in a specific timeframe, separately displaying net and gross orders and more can help the merchant extract relevant accounting information more comfortably.</br>
+Enable merchants to track payments more easily, spot unpaid orders, or quickly evaluate the performance of their shop.
+You can offer notifications for payments that have been pending for too long or monitor single orders. 
+Aggregating the turnover from orders in a specific timeframe, separately displaying net and gross orders and more can help the merchant extract relevant accounting information more comfortably.</br>
 
 A few of the endpoints to achieve this:
 
 | HTTP Verb | Endpoint
 |---|---
-|`GET`|[/orders](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list) 
-|`GET`|[/order/{orderId}](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
-|`GET`|[/orders/{orderId}/processes](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/get-order-processes)
-|`GET`|[/orders/{orderId}/order-note](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-note-update)
+|`GET`|[/orders](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/orders-list) 
+|`GET`|[/order/{orderId}](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/orders/order-get)
```

Comment:
Same issue with the id here and for the next two endpoints..

Reply:
See above comment